tools/actions_local_runner.py

```diff
@@ -21,23 +22,70 @@ class col:
     UNDERLINE = "\033[4m"
 
 
-def color(the_color, text):
+def color(the_color: str, text: str) -> str:
```

Comment:
I see that you've added some type annotations, but you haven't actually added this file to `mypy.ini` or `mypy-strict.ini`; please do that.